middleware/lib/sfMiddleware.ts

```diff
@@ -151,7 +164,7 @@ function createContext(id: string, logger: Logger, secrets: Secrets, reqContext?
  * @param attrVal CloudEvent attribute value to decode.
  * @returns null on empty attrVal, decoded JS Object if successful.
  */
-function decodeCeAttrib(attrVal: string|undefined): any {
+function decodeCeAttrib(attrVal: any): any {
```

Comment:
What causes us to have to use `any`?

Reply:
The latest `cloudevents` sdk does not provide an API like `.getExtensions(key)` any more, so we have to access arbitrary object properties -> so no type safety :-(